### PR TITLE
Update bedrock-mongodb@7.1.0 and remove bedrock-identity

### PR DESCRIPTION
**Pull request by JSAssassin**
Addresses #24 
- All mongodb related calls seems to be implemented correctly.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -136,7 +136,7 @@ api.getRoles = brCallbackify(async actor => {
 /**
  * Adds a new role to the database.
  *
- * @param actor the Identity performing the action.
+ * @param actor the Account performing the action.
```
Comment (aljones15) [inline lib/index.js:139]: good attention to detail.

File: test/mocha/helpers.js [hunk 2]
```diff
@@ -3,34 +3,34 @@
...
 const database = require('bedrock-mongodb');
 const {promisify} = require('util');
 const {util: {uuid}} = require('bedrock');
 
 const api = {};
 module.exports = api;
 
-api.createIdentity = userName => {
-  const newIdentity = {
+api.createAccount = userName => {
```
Comment (aljones15) [inline test/mocha/helpers.js:14]: @mattcollier just to make sure this is correct? We just renaming this `createAccount`? What was this used for before @JSAssassin? DOH! this is a test data function MY BAD.

**Comment by aljones15** [comment]
@JSAssassin is there a branch of veres-wallet and or issuer with the `bedrock-identity`-less setup? Where has this been tested out of the unit tests?

**Comment by JSAssassin** [comment]
> @JSAssassin is there a branch of veres-wallet and or issuer with the `bedrock-identity`-less setup? Where has this been tested out of the unit tests?
@aljones15 yes there are branches on veres-wallet and issuer with `bedrock-identity`-less setup. This one has not been tested there yet. It is just tested out of the unit tests.

**Review by mattcollier — APPROVED** [review]

**Pull request merged**